tests/aws/services/lambda_/test_lambda.py

```diff
@@ -696,6 +697,66 @@ def assert_events():
         )
         snapshot.match("invoke-result-read-number-after-timeout", result)
 
+    @markers.aws.validated
```

Comment:
I'm getting this diff when validating against AWS:

```
>> match key: create-result
	(+) /CreateFunctionResponse/LoggingConfig ( {'LogFormat': 'Text', 'LogGroup': '/aws/lambda/<function-name:1>'} )
```

(Initially wanted to double-check `"_AWS_XRAY_DAEMON_ADDRESS": "169.254.79.129",` but then noticed this one; might be a very recent update)